src/web/modules/services/services.http.ts

```diff
@@ -6,33 +6,47 @@ import { Service, User } from '../../../lib/pay-request/types/adminUsers'
 import { wrapAsyncErrorHandler } from '../../../lib/routes'
 import { sanitiseCustomBrandingURL } from './branding'
 import GatewayAccountRequest from './gatewayAccountRequest.model'
-import { format } from './performancePlatformCsv'
+import { formatPerformancePlatformCsv } from './performancePlatformCsv'
 import { formatErrorsForTemplate, ClientFormError } from '../common/validationErrorFormat'
 import UpdateOrganisationFormRequest from './UpdateOrganisationForm'
 import { IOValidationError } from '../../../lib/errors'
+import { formatServiceExportCsv } from './serviceExportCsv'
 
 function filterRealLiveServices(services: Service[]) {
   return services.filter(service => service.current_go_live_stage === 'LIVE'
     && !service.internal
     && !service.archived)
 }
 
-const overview = async function overview(req: Request, res: Response): Promise<void> {
-  const shouldFilterLiveServices = req.query.live !== "false"
+async function fetchAndFilterServices(shouldFilterLive: boolean): Promise<Service[]> {
   let services: Service[] = await AdminUsers.services()
-  if ( shouldFilterLiveServices ) {
+  if (shouldFilterLive) {
     services = filterRealLiveServices(services)
   }
-  res.render('services/overview', { services, filterLive: shouldFilterLiveServices })
+  return services
+}
+
+const overview = async function overview(req: Request, res: Response): Promise<void> {
+  const shouldFilterLive = req.query.live !== "false"
+  const services = await fetchAndFilterServices(shouldFilterLive)
+  res.render('services/overview', { services, filterLive: shouldFilterLive })
+}
+
+const listCsv = async function exportCsv(req: Request, res: Response): Promise<void> {
+  const shouldFilterLive = req.query.live !== "false"
+  const services = await fetchAndFilterServices(shouldFilterLive)
```

Comment:
I was initially wondering about the error handling for this route but then noticed it's using the wrap controller method on export. That would be a good pattern for us to review and potentially change when we get a chance as it was initially helpful but now done in a few different ways in different parts of the code

Reply:
yeh happy to help out with revisiting that